packages/general/src/crypto/StreamingStandardCrypto.ts

```diff
@@ -0,0 +1,32 @@
+import { Bytes } from "#util/index.js";
+import { Crypto, HashAlgorithm } from "./Crypto.js";
+import { StandardCrypto } from "./StandardCrypto.js";
+
+/**
+ * StandardCrypto implementation that supports streaming hash computation.
+ * Important: It collects chunks from async iterators into memory before computing the hash.
+ *            This is mainly used in testing because of that
```

Comment:
I think it would be OK to put this in the base implementation. Could log a warning there

Reply:
I added and only print a warning when data is more than 100kb ...